docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,130 @@
+The resulting folder structure is similar to:
+
+```
+├── fabric
+  ├── bin
+  │   ├── configtxgen
+  │   ├── configtxlator
+  │   ├── cryptogen
+  │   ├── discover
+  │   ├── idemixgen
+  │   ├── orderer
+  │   └── peer
+  └── config
+    ├── configtx.yaml
+    ├── core.yaml
+    └── orderer.yaml
+```
+
+Along with the relevant binaries, you will receive both the peer binary executable and the peer configuration file, `core.yaml`, that is required to launch a peer on the network. The other files are not required for the peer deployment but will be useful when you attempt to create or edit channels, among other tasks.
+
+**Tip:** Add the location of the peer binary to your `PATH` environment variable so that it can be picked up without fully qualifying the path to the binary executable, for example:
+
+```
+export PATH=<path to download location>/bin:$PATH
+```
+
+After you have mastered deploying and running a peer by using the peer binary and core.yaml configuration file, it is likely that you will want to use a peer container in a Kubernetes or Docker deployment. The Hyperledger Fabric project publishes a [peer image](https://hub.docker.com/r/hyperledger/fabric-peer) that can be used for development and test, and various vendors provide supported peer images. For now though, the purpose of this topic is to teach you how to properly use the peer binary so you can take that knowledge and apply it to the production environment of your choice.
+
+## Prerequisites
+
+Before you can launch a peer node in a production network, you need to make sure you've created and organized the necessary certificates, decided on storage, and configured `core.yaml`.
+
+### Certificates
+
+Note: while **cryptogen** is a convenient utility that can be used to generate certificates for a test network, it should **never** be used on a production network. The core requirement for certificates for Fabric nodes is that they are Elliptic Curve (EC) certificates. You can use any tool you prefer to issue these certificates (for example, OpenSSL). However, the Fabric CA streamlines the process because it generates the Membership Service Providers (MSPs) for you.
+
+Before you can deploy the peer, create the recommended folder structure for the peer certificates that is described in the [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topic to store the generated certificates and MSPs.
+
+This folder structure isn't mandatory, but these instructions presume you have created it:
```

Comment:
Not sure why, but the folder structure looks wonky when I view it at https://hyperledger-fabric--1803.org.readthedocs.build/en/1803/deploypeer/peerdeploy.html